docs/examples/export/export-all-maps.md

```diff
@@ -0,0 +1,88 @@
+```python
```

Comment:
Is there going to be more in this file to describe what the script will be doing?

Reply:
Sorry @andy-esch I'm not sure to understand what do you mean

Reply:
Ah, this is going to be in the developer center's format. I didn't understand how this file was going to be used before.